index.bs

```diff
@@ -807,8 +801,39 @@ The canonicalisation algorithm is as follows:
 
 This canonicalisation algorithm is very similar to that used in [[JCS]], except that we stipulate an ordering of keys, and we enforce upper-case rather than lower-case hex digits.
 
+<div class="example">
+Given an item two fields <code>foo</code> with a value <code>abc</code> and
```

Comment:
Perhaps something like:

> As an example, take an item with two fields <code>foo</code> and <code>bar</code> with values <code>abc</code> and <code>xyz</code> respectively.  This can be expressed as JSON: